src/semantic-ui/site/globals/site.overrides

```diff
@@ -215,6 +215,24 @@ a {
   }
 }
 
+.custom-label {
```

Comment:
I fear that this is too generic, while it is used in a very specific context.
Is this re-usable anywhere else? If not:
```
.<doc req form class> {
    .custom-label
}
```
or even better prefer ids `#` whenever possible, the rendering is faster.

If it is re-usable, then I would call it with some more meaningful names.